src/components/composer/composer/composer.ts

```diff
@@ -309,8 +309,8 @@ export class Composer extends Component<Props, SpreadsheetChildEnv> {
     } else {
       ev.stopPropagation();
     }
-    const { start, end } = this.contentHelper.getCurrentSelection();
-    if (!this.env.model.getters.isSelectingForComposer()) {
+    if (!this.env.model.getters.isSelectingForComposer() && !["Enter", "Tab"].includes(ev.key)) {
```

Comment:
this will break the multi line inputs starting saas-16.1. If I'm not mistaken,  the specificity of Enter and Tab is that they stop the edition inside their handlers, so we might just have to check that we are not  in selection mode and we are still editing, Since there are only two modes of edition, we can just check that 
```typescript
this.env.model.getters.editionMode() === "editing"
```

Reply:
>  the specificity of Enter and Tab is that they stop the edition inside their handlers

not when their effect is to autocomplete.